### PR TITLE
Added interface for Customer Management

### PR DESCRIPTION
**Pull request by TanjaSvard**
Added interface for Customer Management

### CODE REVIEW

File: ZupahBank/BusinessLib/Interfaces/ICustomerManagement.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLib.Interfaces
+{
+    interface ICustomerManagement
+    {
+        void Create(string name, string legalId);
+        void Delete();
```
Comment (DannyBoyIT) [inline ZupahBank/BusinessLib/Interfaces/ICustomerManagement.cs]: Which object will be deleted here? Maybe the method should have some input parameters.

File: ZupahBank/BusinessLib/Interfaces/ICustomerManagement.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        Customer Edit();
```
Comment (DannyBoyIT) [inline ZupahBank/BusinessLib/Interfaces/ICustomerManagement.cs]: Same as above

**Review by DannyBoyIT — CHANGES_REQUESTED** [review]
Check the comments! ;)

**Comment by TanjaSvard** [comment]
Fixed parameters

**Review by DannyBoyIT — CHANGES_REQUESTED** [review]
Change legal id to customerId

**Review by DannyBoyIT — APPROVED** [review]

**Pull request merged**